### DISCUSSION TITLE
Webhooks - how to create it, or know it is created, for Stripe

### DISCUSSION THREAD

**Discussion by amityweb** [saleor/saleor#15914] [General]
I have read the webhook for Stripe is created automatically by Saleor. How can I find out if this is actually happening? Stripe reports no Webhook attempts at all. 

There is a field to enter a webhook endpoint but that is disabled. 

Is there a known endpoint I can use to manually create the webhook in Stripe? 

I have read Saleor will not create a webhook if the site domain is localhost so I have updated it to the URL of Saleor core using mutation and thats change it, but webhooks still now working. 

Stripe is in test mode, using test keys, and shows the order payment successful in there. 

Thanks

**Comment by russzin6** [comment]
hi , I don't understand how the complete payment process of saleor's webhook and stripe works. Can anyone explain it to me?
Thanks！

**Comment by aniav** [comment]
Hey, we have mostly moved our discussions [to Discord](https://discord.com/invite/H52JTZAtSH) and there actually were a few threads about Stripe and solutions to common problems. We will be slowly closing off discussions on github. They have proven to be not dynamic enough 🙌
